src/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/Resources.resx

```diff
@@ -120,6 +120,30 @@
   <data name="ArgumentCannotBeNullOrEmpy" xml:space="preserve">
     <value>Value cannot be null or empty.</value>
   </data>
+  <data name="InjectDirective_Description" xml:space="preserve">
+    <value>Declare a property and inject a service from the application's service container into it.</value>
+  </data>
+  <data name="InjectDirective_MemberToken_Description" xml:space="preserve">
+    <value>The name of the variable to inject the type in.</value>
```

Comment:
@DamianEdwards, I mostly used fillers for these token names and descriptions. Could you go through the resx files in this PR and comment on the ones that need to change?